src/olympia/zadmin/tasks.py

```diff
@@ -587,14 +587,30 @@ def fetch_langpack(url, xpi, **kw):
             log.info('[@None] Created new "{0}" language pack, version {1}'
                      .format(xpi, data['version']))
 
+        # Set the category
+        for app in version.compatible_apps:
+            static_category = (
+                CATEGORIES[app.id][amo.ADDON_LPAPP]['general'])
+            category, _ = Category.objects.get_or_create(
+                id=static_category.id, defaults=static_category.__dict__)
+            AddonCategory.objects.get_or_create(addon=addon, category=category)
+        del addon.all_categories
```

Comment:
Let's add a comment like `# Remove old, outdated categories cache on the model.` or so. It's not immediately clear that `all_categories` is just simply a cache.